packages/replayio/src/commands/record.ts

```diff
@@ -176,3 +171,57 @@ async function record(url: string = "about:blank") {
 
   await exitProcess(0);
 }
+
+async function reportRecorderCrash(stderr: string) {
+  console.log("\nSomething went wrong while recording. Try again.");
+
+  const errorLogPath = getReplayPath("recorder-crash.log");
+  writeFileSync(errorLogPath, stderr, "utf8");
+  console.log(dim(`More information can be found in ${errorLogPath}`));
+
+  const accessToken = await checkAuthentication();
+
+  if (!accessToken) {
+    return;
+  }
+
+  const userAgent = getUserAgent();
+
+  const formData = new FormData();
+
+  formData.set("buildId", getCurrentRuntimeMetadata("chromium")?.buildId ?? "unknown");
+  formData.set("createdAt", new Date().toISOString());
+  formData.set("userAgent", userAgent);
+
+  formData.append(
+    "log",
+    new File([await readFile(errorLogPath)], "log.txt", { type: "text/plain" })
+  );
+
+  const latestCrashpad =
+    runtimeMetadata.crashpadDirectory &&
+    (await findMostRecentFile(runtimeMetadata.crashpadDirectory, fileName =>
+      fileName.endsWith(".dmp")
+    ));
+  if (latestCrashpad) {
+    formData.append(
+      "crashpad",
+      new File([await readFile(latestCrashpad)], "crash.dmp", { type: "application/octet-stream" })
+    );
+  }
+
+  try {
+    const response = await fetch(`${replayApiServer}/v1/browser-crash`, {
+      method: "POST",
+      headers: {
+        Authorization: `Bearer ${accessToken}`,
+        "User-Agent": userAgent,
+      },
+      body: formData,
+    });
+    if (response.status >= 200 && response.status < 300) {
+      console.log(dim(`The crash was reported to the Replay team`));
+      return;
+    }
+  } catch (err) {}
```

Comment:
nit: Should we print the error to the `DEBUG` output?

Reply:
ye, good idea - i added this just now